testapps/testapp/build.gradle

```diff
@@ -50,10 +55,11 @@ android {
     defaultConfig {
         multiDexEnabled true
         applicationId "com.msft.identity.client.sample"
-        minSdkVersion rootProject.ext.minSdkVersion
+        minSdkVersion 24
```

Comment:
Should we create a variable for this? I think Sowmya pushed a similar change recently and had created a variable with this minSdkVersion, I don't remember if it's the same scenario though

Reply:
Yes, I created this variable : brokerProjectMinSdkVersion. May be we create similar one for MSAL

Reply:
Let me handle version numbers in a separate PR.